_includes/templates/nav/primary-nav.html

```diff
@@ -0,0 +1,25 @@
+{% import "templates/nav/_vars-primary-nav.html" as vars with context %}
+<nav class="primary-nav sliding-nav_nav">
+    <ul class="primary-nav_top-level-list
+               wrapper
+               list__unstyled
+               list__horizontal
+               list-expanding">
+        {%- for nav_title, sub_nav_title, sub_nav_media, sub_nav_items in vars.nav_items %}
+            <li class="list_item list-expanding_list-item">
+                {%- if sub_nav_items|length > 0 %}
+                    {# TODO: Make this a <button>, it is not a real link. #}
+                    <a class="list_link list-expanding_trigger" href="#">
+                        {{ nav_title }}
+                    </a>
+                    {% import 'macros/sub-nav.html' as subNav %}
```

Comment:
`sub-nav.html` -> `subNav` We typically haven't changed the name when importing a macro, any reason to here?